### DISCUSSION TITLE
How to hide my number from the status bar when using nchat with multiple profiles for the same protocol?

### DISCUSSION THREAD

**Discussion by sachinsenal0x64** [d99kris/nchat#157] [Q&A]
![283016774-38e25610-f77b-4b21-b3a5-ce1449b56afe](https://github.com/d99kris/nchat/assets/127573781/ca13cc45-e475-4c35-9925-bc9a00253b68)

**Comment by d99kris** [comment]
Hi @sachinsenal0x64 - if you have multiple Telegram or WhatsApp accounts setup, nchat displays which one the current chat conversation belongs to by suffixing the protocol name with your phone number, i.e. like `Telegram_+1555123488999`.

You can edit `~/.nchat/profiles/Telegram_+nnnnn/telegram.conf` and set a custom `profile_display_name`, for example `profile_display_name=Telegram_Home` or `profile_display_name=Telegram_Work`, etc.

Reference: [profile_display_name](https://github.com/d99kris/nchat#profile_display_name)

**Reply by sachinsenal0x64** [reply]
Thank you for help :)

**Comment by sachinsenal0x64** [comment]
> Hi @sachinsenal0x64 - if you have multiple Telegram or WhatsApp accounts setup, nchat displays which one the current chat conversation belongs to by suffixing the protocol name with your phone number, i.e. like `Telegram_+1555123488999`.
> 
> You can edit `~/.nchat/profiles/Telegram_+nnnnn/telegram.conf` and set a custom `profile_display_name`, for example `profile_display_name=Telegram_Home` or `profile_display_name=Telegram_Work`, etc.
> 
> Reference: [profile_display_name](https://github.com/d99kris/nchat#profile_display_name)
